### PR TITLE
Syndicate Duffelbag Rerework

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Syndicate duffelbags can fit 2 extra bulky items, down from three.

Reduced syndicate duffelbag's unzipped slowdown from '1' to '0.3', and set its zipping-up sped to 0.5, same as unzipping.

Added the following items to the Syndicate Duffelbag bulky exception list: Greentext, mech removal tool, gibtonite, skub, golem shells, mech ammo. Roughly sorted the list by item category.

Fixed the syndie surgery duffelbag having more items than it can hold by removing the redundant surgical drill (Upgraded cauteries can turn into one anyways)

Any storage item with a can_hold description can be examined twice to see what it can hold now.

## Why It's Good For The Game

> Syndicate duffelbags can fit 2 extra bulky items, down from three.

> Reduced syndicate duffelbag's unzipped slowdown from '1' to '0.3', and set its zipping-up sped to 0.5, same as unzipping.

For most intents and purposes, it seems the syndicate duffelbag has gone from 'bland upgrade to backpack', to 'useless'. This is especially made apparent because it isn't exactly shown to the player that these duffelbags can carry bulky items (I didn't even know about it until I was making this PR!)

The extra bulky item hold concept is great, but I have my issues with the item as-is that I seek to fix with this PR. There are TONS of issues with being unable to access your bag quickly, which is twice as relevant when your bag is an incredibly conspicious traitor item. Sure, you can have it in your hand, but then why even have it in the first place?

That's why I want to reduce the slowdown significantly. '1' slowdowns are thrown around the whole game like they're reasonable (galoshes, water back-tanks, biosuits) - they aren't. '1' slowdown is CRIPPLING. It makes you frustratingly slow and effectively destroys any combat maneuvering you can do. This is very relevant for a traitorious item.

The zip speed helps one use the duffelbag as a storage item dynamically, letting the item be an actual trade-off rather than mostly a downside. Gives you a reason to use it rather than just buying a smuggler satchel for more storage.

Of course these are some hefty buffs, so I lowered the bulky storage to make up for it. I can bring it back up to 3 if wanted.

> Added the following items to the Syndicate Duffelbag bulky exception list: Greentext, mech removal tool, gibtonite, skub, golem shells, mech ammo. Roughly sorted the list by item category.

Some traitorious items that felt like they should be allowed in. Honestly, I think this shouldn't even be an exception hold except for blacklisting clearly bonkers things like backpacks, but whatevs.

> Any storage item with a can_hold description can be examined twice to see what it can hold now.

Generalization is awesome. Hardcoding is cringe!

## Changelog

:cl:
balance: Syndicate duffelbags can fit 2 extra bulky items, down from three.
balance: Reduced syndicate duffelbag's unzipped slowdown from '1' to '0.3', and set its zipping-up sped to 0.5, same as unzipping.
add: Added the following items to the Syndicate Duffelbag bulky exception list: Greentext, mech removal tool, gibtonite, skub, golem shells, mech ammo. Roughly sorted the list by item category.
fix: Fixed the syndie surgery duffelbag having more items than it can hold by removing the redundant surgical drill (Upgraded cauteries can turn into one anyways)
qol: Any storage item with a can_hold description can be examined twice to see what it can hold now.
fix: The parent crayon's name is 'crayon' to prevent any weirdness with things that show the parent type's name.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
image

why do we have a crayon naming unit test. wtf.

**Comment by carlarctg** [comment]
oh no i sumoned hte moth

**Comment by SomeoneYouProbablyKnow** [comment]
SKUB

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -72,13 +72,15 @@
 	user.suicide_log()
 	qdel(user)
 
+
+	storage_type = /datum/storage/backpack/santabag
```
Comment (Jacquerel) [inline code/game/objects/items/storage/backpack.dm:76]: This change looks incorrect

**Review by Jacquerel — CHANGES_REQUESTED** [review]

**Pull request merged**